server/utils/assessmentUtils.ts

```diff
@@ -133,6 +133,16 @@ const formatDays = (days: number): string => {
   return `${days} Day${days > 1 ? 's' : ''}`
 }
 
+const formatDaysUntilDueWithWarning = (assessment: Assessment): string => {
+  const days = daysUntilDue(assessment)
+  if (days < DUE_DATE_APPROACHING_DAYS_WINDOW) {
+    return `<strong class="assessments--index__warning">${formatDays(
+      days,
+    )}<span class="govuk-visually-hidden">(Approaching due date)</span></strong>`
```

Comment:
Do we want a space before the <span> tag, so it's not "4 Days(Approaching due date)"?